integration/js/pages/AppPage.js

```diff
@@ -551,7 +567,11 @@ class AppPage {
   }
 
   async isAuthenticating() {
-    return await this.driver.findElement(elements.authenticationFlow).isDisplayed();
+    return await this.driver.findElement(elements.authenticationFlow).isDisplayed() || await this.driver.findElement(elements.permisssionsFlow).isDisplayed();
+  }
+  
+  async isWaitingForPageToLoad() {
+    return await this.driver.findElements(elements.authenticationFlow).length > 0;
   }
```

Comment:
Please add a method description here. This method is testing whether the page is loaded via checking HTML element length. In the description, please explain why a `length > 0` means the page is loaded? It is not very intuitive for users who are not familiar with the mobile stack.

Reply:
I think just a better method name would be great. Can we specify which page this is (e.g., isiWatingForAuthPageToLoad)?
Also why can't we use isDisplayed() here instead of checking for length?

Reply:
Will add. It's not actually checking the length of the element or anything with the mobile stack, it's just checking if a element from our browser demo exists so it won't try to run the rest of the test while it's on the default page, only a problem with IOS Safari though.

Reply:
@ltrung Yeah I'll change to isWaitingForAuthPageToLoad, can't use isDisplayed because it's not actually finding the element it's checking if it can find the element. findElement produces a error if it can't find the element and there's no selenium function for doesElementExist so to find if the element exists I use findElements and see if it found any that exist.